pkg/api/volume/formatter.go

```diff
@@ -0,0 +1,83 @@
+package volume
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/rancher/apiserver/pkg/apierror"
+	"github.com/rancher/apiserver/pkg/types"
+	v1 "github.com/rancher/wrangler/pkg/generated/controllers/core/v1"
+	"github.com/rancher/wrangler/pkg/schemas/validation"
+	"github.com/sirupsen/logrus"
+
+	harvesterv1 "github.com/harvester/harvester/pkg/apis/harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester/pkg/generated/controllers/harvesterhci.io/v1beta1"
+)
+
+const (
+	actionExport = "export"
+)
+
+func Formatter(request *types.APIRequest, resource *types.RawResource) {
+	resource.Actions = make(map[string]string, 1)
+	if resource.APIObject.Data().String("spec", "sourceType") == harvesterv1.VirtualMachineImageSourceTypeExportVolume {
+		resource.AddAction(request, actionExport)
+	}
```

Comment:
I think we need to add the action to volume(pvc) resources. Looks like we are adding it to vmimages here

Reply:
@gitlawr updated